### ISSUE TITLE
Some examples do not return expected output

### ISSUE THREAD

**Issue by ojundt** [peppolautoriteit-nl/testset#4]
Hi,

I just applied the SI-UBL-INV.xsl from the [1.1 release in the validation repo](https://github.com/SimplerInvoicing/validation/blob/691423ae103146c4041cf45b7a5182dad1248270/SI-UBL-INV.xsl) to all 1.1 examples and all return the expected output except the following two:

`SI-UBL-1.1/SI-UBL-1.1-ok-rounding-vat-1.xml` gives:
`[EUGEN-T10-R043]-The total tax amount MUST equal the sum of tax amounts per category.`

`SI-UBL-1.1/SI-UBL-1.1-ok-extension.xml` gives:
`[SI-V11-INV-R026]-Currency Identifier MUST be stated in the currency as defined on header level`

**Comment by ojundt** [comment]
Just found another 3. These error examples trigger no error or warning:
```
SI-UBL-1.1/SI-UBL-1.1-error-corrective-billingreference.xml
SI-UBL-1.1/SI-UBL-1.1-error-BII2-T10-R032.xml
SI-UBL-1.1/SI-UBL-1.1-error-BII2-T10-R058.xml
```

**Comment by ojundt** [comment]
And more, this time for 1.2:

```
SI-UBL-1.2/SI-UBL-INV-1.2-warning-CL-T10-R007.xml
SI-UBL-1.2/SI-UBL-INV-1.2-warning-CL-T10-R006.xml
```
both trigger a fatal error instead of a warning

```
SI-UBL-1.2/SI-UBL-INV-1.2-error-CL-T10-R006.xml
SI-UBL-1.2/SI-UBL-INV-1.2-error-corrective-billingreference.xml
SI-UBL-1.2/SI-UBL-INV-1.2-error-CL-T10-R010.xml
```
trigger no warning or error (except for `CL-T10-R010` which returns a warning).

**Comment by ojundt** [comment]
@tjeb any chance to take a look at this?

**Comment by tjeb** [comment]
This has been on the just-out-of-reach end of my todo list way to long, but I hope better late than never: I have taken a look at this, and fixed or removed a number of documents (as well as some validation rules).

There have been some other updates to the xslt's in the meantime (new peppol schematrons), so maybe some issues were stealth-fixed by the validation files.

1.1:
SI-UBL-1.1/SI-UBL-1.1-ok-rounding-vat-1.xml: It shows no error here, is this still an issue?
SI-UBL-1.1/SI-UBL-1.1-ok-extension.xml: same for this one
SI-UBL-1.1/SI-UBL-1.1-error-corrective-billingreference.xml: This file was wrong, but the 1.1 check was as well, fixed both (backported validation check from 1.2, removed the BillingReference in this file). Renamed file to 
SI-UBL-1.1/SI-UBL-1.1-error-BII2-T10-R032.xml: This was actually an error in the validation rules, due to it being based on an older version of PEPPOL. I have backported the fix in the validation rules.
SI-UBL-1.1/SI-UBL-1.1-error-BII2-T10-R058.xml: Same here.

1.2:

SI-UBL-1.2/SI-UBL-INV-1.2-error-corrective-billingreference.xml: Fixed (removed the billing reference so it errors now). Renamed to SI-UBL-INV-1.2-error-SI-INV-R453.xml
SI-UBL-1.2/SI-UBL-INV-1.2-error-CL-T10-R006.xml: Changed the code to one that is actually wrong.
SI-UBL-1.2/SI-UBL-INV-1.2-error-CL-T10-R010.xml: This should be a warning (and the warning version of that file was already present). Removed this document.
